packages/pianobar/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://6xq.net/pianobar/
+TERMUX_PKG_DESCRIPTION="pianobar is a free/open-source, console-based client for the personalized online radio Pandora."
+TERMUX_PKG_VERSION=2018.06.22
+TERMUX_PKG_REVISION=1
```

Comment:
No need to set TERMUX_PKG_REVISION for a new package (and counter starts from 0).